core/lib/instrument_api.h

```diff
@@ -6201,6 +6201,13 @@ DR_API
 bool
 dr_prepopulate_cache(app_pc *tags, size_t tags_count);
 
+DR_API
+/**
+ * Get the number of blocks built so far, globally. The API is not thread-safe.
+ */
+uint64
+dr_stats_get_built_blocks_count();
```

Comment:
Hmm, if the plan is to provide a set of statistics that might grow to a dozen or more, maybe it is better to return a pointer to a struct that we can extend, making it much easier to add new statistics without having to add a whole new API function?

Reply:
Done.